.github/workflows/r_cmd_check.yaml

```diff
@@ -1,36 +1,41 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
```

Comment:
Copied in the more standard CI R checks from the hubverse. I didn't include the entire matrix of r versions and operating systems and propose to punt on that until we've [fixed up the test suite.](https://github.com/reichlab/covidHubUtils/issues/411)